bimmer_connected/state.py

```diff
@@ -125,12 +125,17 @@ def update_data(self) -> None:
             'dlat': self._vehicle.observer_latitude,
             'dlon': self._vehicle.observer_longitude,
         }
-        response = self._account.send_request(
-            VEHICLE_STATUS_URL.format(server=self._account.server_url, vin=self._vehicle.vin), logfilename='status',
-            params=params)
-        attributes = response.json()['vehicleStatus']
+        # Pre-NBT vehicles don't seem to have a status that can be reported
+        try:
+            response = self._account.send_request(
+                VEHICLE_STATUS_URL.format(server=self._account.server_url, vin=self._vehicle.vin), logfilename='status',
+                params=params)
+            attributes = response.json()['vehicleStatus']
+            _LOGGER.debug('received new data from connected drive')
+        except OSError:
+            attributes = {}
+            _LOGGER.debug('Unable to retrieve vehicle status from connected drive')
```

Comment:
I suggest to add some more explanation to this log line to make clear that a reason this is failing also is that the car is too old to be supported by the integration.

Reply:
And should we merge this into dev? Then you can also test the beta release for US cars?